### PR TITLE
ADD: Number should be formatted to be a decimal

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Review by Overtorment — APPROVED** [review]

File: ios/Shared/MarketData.swift [hunk 1]
```diff
@@ -14,7 +14,18 @@ struct MarketData:Codable  {
...
   var formattedNextBlock: String {
-    return nextBlock == "..." ? "..." : #"\#(nextBlock) sat/b"#
+    if nextBlock == "..." {
+      return "..."
+    } else {
+      if let nextBlockInt = Int(nextBlock) {
+        let numberFormatter = NumberFormatter()
+        numberFormatter.numberStyle = .decimal
+        if let formattedNumber = numberFormatter.string(from: NSNumber(value: nextBlockInt)) {
+          return "\(formattedNumber) sat/b"
```
Comment (Overtorment) [inline ios/Shared/MarketData.swift:24]: Btw you might want to write sat/vb
(Virtual byte)

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/obmdnggeto3gpg3gd75bt45ucu?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1297)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/oogu7qctnbucqmh2veqxn5rcba?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1298)

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**